modules/ob-installing-buildah-strategy.adoc

```diff
@@ -12,12 +12,14 @@ You can install the `buildah` cluster build strategy, which does not require a d
 .Prerequisites
 
 * You have installed the `oc` CLI.
+* You have installed the {builds-operator}. For more information, see xref:../installing/installing-openshift-builds#installing-builds-with-operator_installing-openshift-builds[Installing {builds-shortname} by using the web console].
```

Comment:
Referenced in [the doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references), we cannot have xrefs in modules. 
You can add the xrefs to a .Additional resources section in a assembly, but they cannot be in modules.